src/main/java/se/kth/sda/skeleton/user/UserController.java

```diff
@@ -49,4 +51,27 @@ public UserController(UserRepository userRepository, AuthService authService) {
         return ResponseEntity.ok(updatedUser);
 
     }
+
+    /**
+     * Update user's accountType for logged in user
+     *
+     * @param currentUser the user description
+     * @return HTTP ok status and the updated user
+     */
+    @PutMapping("/users/edit")
+    public ResponseEntity<User> updateUserAccountType(@RequestBody User currentUser) {
+        String email = authService.getLoggedInUserEmail();
+        User user = userRepository.findByEmail(email);
+        String accountType = currentUser.getAccountType();
+        user.setAccountType(accountType);
+        if (user.getAccountType().equals("human") ||
```

Comment:
This part shares the same codes as _register method in UserService.java_. Would it be possible for you to refactor the codes into a method and reuse it here?

Reply:
yeah, that's a good idea